### PR TITLE
Adding Home page to navigation in Twig

### PR DESCRIPTION
**Pull request by NORDCOMPCZ**
Given example is PHP only, adding working Twig one

  
  

## Changes in this pull request  
Resolves #

### CODE REVIEW

File: doc/Development_Documentation/03_Documents/03_Navigation.md [hunk 1]
```diff
@@ -487,6 +487,15 @@ $navigation->addPage([
...
 ]);
 ```
+```twig
+ {% set mainNavigation = mainNavigation.addPage({
+                            'order': -1,
+                            'uri': navStartNode.url,
+                            'label': 'Home',
+                            'title': "Home"
+                        })
+%}
```
Comment (jdreesen) [inline doc/Development_Documentation/03_Documents/03_Navigation.md:497]: There's no need in re-assigning the variable, so this should be sufficient:

```suggestion
{{ mainNavigation.addPage({
    order: -1, // put it in front of all the others
    uri: '/', //path to homepage
    label: 'Home', //visible label
    title: 'Homepage', //tooltip text
}) }}
```

Comment (NORDCOMPCZ) [reply doc/Development_Documentation/03_Documents/03_Navigation.md:497]: @jdreesen unfortunately not. Your suggested change was my first shot too but this construction ends up with `Object of class Pimcore\Navigation\Container could not be converted to string` error. Therefore the re-assignment.

Comment (dpfaffenbauer) [reply doc/Development_Documentation/03_Documents/03_Navigation.md:497]: `{% do mainNavigation.addPage.... %}`

Comment (NORDCOMPCZ) [reply doc/Development_Documentation/03_Documents/03_Navigation.md]: @dpfaffenbauer you're right, the`do` is the correct  solution here

File: doc/Development_Documentation/03_Documents/03_Navigation.md [hunk 2]
```diff
@@ -487,6 +487,15 @@ $navigation->addPage([
...
 ]);
 ```
+```twig
+ {% do mainNavigation.addPage({
+                            'order': -1,
+                            'uri': navStartNode.url,
+                            'label': 'Home',
+                            'title': "Home"
+                        })
+%}
```
Comment (dvesh3) [inline doc/Development_Documentation/03_Documents/03_Navigation.md:497]:
```suggestion
 {% do navigation.addPage({
                            'order': -1,
                            'uri': '/',
                            'label': 'Home',
                            'title': 'Homepage'
                        })
%}
```

Comment (dvesh3) [reply doc/Development_Documentation/03_Documents/03_Navigation.md:497]: and please wrap them into a code-section container, like here: https://github.com/pimcore/pimcore/commit/19f4e0b9cf0713bfb55ff577b8c2c7e404f78d2c thanks!

Comment (NORDCOMPCZ) [reply doc/Development_Documentation/03_Documents/03_Navigation.md:497]: @dvesh3 the `'uri': '/'`is probably not the best solution when you have multiple languages...

Comment (dvesh3) [reply doc/Development_Documentation/03_Documents/03_Navigation.md:497]: well, this is just a basic example and it does not talk about multiple languages setup here.

Comment (dvesh3) [reply doc/Development_Documentation/03_Documents/03_Navigation.md:497]: @NORDCOMPCZ 🏓

**Review by dvesh3 — CHANGES_REQUESTED** [review]
please see https://github.com/pimcore/pimcore/pull/6896#discussion_r467744961 & https://github.com/pimcore/pimcore/pull/6896#discussion_r467745668

File: doc/Development_Documentation/03_Documents/03_Navigation.md [hunk 3]
```diff
@@ -487,6 +487,15 @@ $navigation->addPage([
    'title' => 'Homepage' //tooltip text
 ]);
 ```
+```twig
+ {% do mainNavigation.addPage({
+                            'order': -1,
+                            'uri': navStartNode.url,
+                            'label': 'Home',
+                            'title': "Home"
```
Comment (NiklasBr) [inline doc/Development_Documentation/03_Documents/03_Navigation.md:495]:
```suggestion
                            'label': 'Home'|trans,
                            'title': 'Home'|trans,
```

File: doc/Development_Documentation/03_Documents/03_Navigation.md [hunk 4]
```diff
@@ -487,6 +487,15 @@ $navigation->addPage([
    'title' => 'Homepage' //tooltip text
 ]);
 ```
+```twig
+ {% do mainNavigation.addPage({
+                            'order': -1,
+                            'uri': navStartNode.url,
```
Comment (brusch) [inline doc/Development_Documentation/03_Documents/03_Navigation.md:493]:
```suggestion
                            'uri': '/',
```

File: doc/Development_Documentation/03_Documents/03_Navigation.md [hunk 5]
```diff
@@ -487,6 +487,15 @@ $navigation->addPage([
    'title' => 'Homepage' //tooltip text
 ]);
 ```
+```twig
+ {% do mainNavigation.addPage({
```
Comment (brusch) [inline doc/Development_Documentation/03_Documents/03_Navigation.md:491]:
```suggestion
 {% do navigation.addPage({
```

**Comment by jdreesen** [comment]
Oh noes! It has no code-section/tabs -> #7029

**Pull request merged**